backends/dpdk/dpdkContext.cpp

```diff
@@ -210,6 +210,9 @@ size_t DpdkContextGenerator::getHandleId(cstring name) {
         if (x.first.find(name.c_str())) {
             id = context_handle_map[x.first];
             break;
+        } else if (name.find(x.first.c_str()) && name.endsWith("_sel")) {
```

Comment:
We are creating new ID here. This leaves different Selector table ids in context json and bfrt.json.